### PR TITLE
Fixed HDFS plugin

### PR DESCRIPTION
**Pull request by matriv**

### CODE REVIEW

File: blackbox/hdfs/src/tests.py [hunk 1]
```diff
@@ -47,15 +47,16 @@
 
 hdfs_repo_zip = os.path.join(
```
Comment (mfussenegger) [inline blackbox/hdfs/src/tests.py]: filename doesn't fit anymore

File: es/es-repository-hdfs/build.gradle [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+dependencies {
+    compile project(':es:es-core')
+    compile "org.apache.hadoop:hadoop-client:${versions.hadoop2}"
+    compile "org.apache.hadoop:hadoop-common:${versions.hadoop2}"
+    compile "org.apache.hadoop:hadoop-annotations:${versions.hadoop2}"
+    compile "org.apache.hadoop:hadoop-auth:${versions.hadoop2}"
+    compile "org.apache.hadoop:hadoop-hdfs:${versions.hadoop2}"
+    compile 'org.apache.htrace:htrace-core:3.1.0-incubating'
+    compile 'com.google.guava:guava:16.0.1'
```
Comment (mfussenegger) [inline es/es-repository-hdfs/build.gradle]: these are old versions, can't we use newer ones here?

File: gradle/version.properties [hunk 3]
```diff
@@ -5,21 +5,29 @@ jodatime=2.9.5
...
+# ES HDFS
+hadoop2=2.7.1
+apache_log4j=2.6.2
+
 # ES optional
 spatial4j=0.6
 jts=1.13
 jackson=2.8.1
-log4j=2.6.2
+log4j=1.2.7
```
Comment (mfussenegger) [inline gradle/version.properties]: can't we keep log4j 2?

File: es/es-repository-hdfs/build.gradle [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+def classname = 'org.elasticsearch.repositories.hdfs.HdfsPlugin'
+def plugin_repositoryhdfs_version = '1.0.0'
+
+configurations {
+    extraLibs.transitive = false
+}
+
+dependencies {
+    compile project(':es:es-core')
+    compile "org.apache.hadoop:hadoop-client:${versions.hadoop2}"
```
Comment (seut) [inline es/es-repository-hdfs/build.gradle]: we are now requiring all the hadoop deps? we should avoid that and as all are defined under the extraLib configuration it is not needed afail

File: es/es-repository-hdfs/build.gradle [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+        ant.entry(key: 'version', value: plugin_repositoryhdfs_version)
+        ant.entry(key: 'site', value: true)
+        ant.entry(key: 'elasticsearch.version', value: "${versions.elasticsearch}")
+        ant.entry(key: 'java.version', value: project.targetCompatibility)
+        ant.entry(key: 'classname', value: classname)
+    }
+}
+
+task copyToLib(type: Copy) {
+    into buildDir.path + '/lib'
```
Comment (seut) [inline es/es-repository-hdfs/build.gradle]: we dont want to ship crate with all hadoop libs. if the user wants to use it, it have to copy the libs by himself as the current documentation also states

**Pull request merged**